CLR/Libraries/Windows_Devices/windows_devices_stub.cpp

```diff
@@ -59,3 +59,31 @@ HRESULT Library_windows_devices_native_Windows_Devices_Gpio_GpioPin::SetDriveMod
     TINYCLR_SET_AND_LEAVE(CLR_E_NOT_SUPPORTED);
     TINYCLR_NOCLEANUP();
 }
+
+HRESULT Library_windows_devices_native_Windows_Devices_Spi_SpiBusInfo::get_ChipSelectLineCount___I4( CLR_RT_StackFrame& stack )
+{
+    TINYCLR_HEADER();
+    TINYCLR_SET_AND_LEAVE(CLR_E_NOT_SUPPORTED);
+    TINYCLR_NOCLEANUP();

```

Comment:
I was using this earlier, but Steve asked me to switch to not supported... I'll switch it back.